routes/subscriptions/SubscriptionController.cs

```diff
@@ -0,0 +1,90 @@
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using TAIBackend.Model;
+
+namespace TAIBackend.routes.subscriptions;
+
+[Route("subscriptions")]
+public class SubscriptionController : Controller
+{
+    [Authorize(AuthenticationSchemes = CookieAuthenticationDefaults.AuthenticationScheme)]
+    [HttpGet("me")]
+    public async Task<IActionResult> GetSubscibedAccounts(YoutubeContext db)
+    {
+        try
+        {
+            var ownerAccountId = Int64.Parse(User.FindFirst(ClaimTypes.NameIdentifier)!.Value);
+            var subs = await db.Subscriptions.Where(s => s.Owneraccountid == ownerAccountId).ToListAsync();
+            return Ok(new
+            {
+                data = subs.ToArray().Select(s => new
+                {
+                    id = s.Id, subscribedId = s.Owneraccountid
+                }),
+                count = subs.Count
```

Comment:
to nie jest mi potrzebne, ponieważ tutaj nie ma i tak paginacji. Zwracane dane to powinna być tablica obiektów
z polami userId - Id użytkownika którego subskrybujemy i userFullName -żeby wyświetlić nazwę użytkownika którego subskrybujemy

Reply:
![image](https://github.com/Projekt-TAI/youtube/assets/56255905/b47173e3-58b4-41b8-8f96-b5b80c5d4d97)